Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -1,17 +1,20 @@
 import UIKit
+import Checkout
 
 protocol PaymentViewControllerDelegate: AnyObject {
     func addBillingButtonIsPressed(sender: UINavigationController?)
     func editBillingButtonIsPressed(sender: UINavigationController?)
+    func expiryDateIsUpdated(value: ExpiryDate)
 }
 
-final class PaymentViewController: UIViewController {
+final class PaymentViewController: UIViewController{
 
     //MARK: - Variables
 
     weak var delegate: PaymentViewControllerDelegate?
+    
     private(set) var viewModel: PaymentViewModel
-
+    private var notificationCenter = NotificationCenter.default
```

Comment:
type should be a protocol. this should also passed from init so we can mock it for testing

Reply:
It will be updated when other components are added to PaymentViewController.